feature-examples/python/api_key/api_key_main.py

```diff
@@ -0,0 +1,78 @@
+"""Main file for Vectara feature examples related to ApiKey."""
+import argparse
+import logging
+import sys
+from typing import Optional
+
+from api_key import create_api_key
+from api_key import delete_api_key
+from api_key import enable_api_key
+from api_key import list_api_key
+from utils import utils
+
+
+def main() -> None:
+    """Main function."""
+    parser = argparse.ArgumentParser(
+        description="Vectara example to perform different operations related to API keys."
+    )
+
+    parser.add_argument(
+        "--customer-id",
+        type=int,
+        required=True,
+        help="Unique customer ID in Vectara platform.",
+    )
+    parser.add_argument(
+        "--corpus-id",
+        type=int,
+        required=True,
+        help="Corpus ID to which API key will be created.",
+    )
+    parser.add_argument(
+        "--app-client-id",
+        required=True,
+        help="This app client should have enough rights.",
+    )
+    parser.add_argument("--app-client-secret", required=True)
+    parser.add_argument(
+        "--auth-url", required=True, help="The auth url for this customer."
+    )
+
+    args = parser.parse_args()
+
+    jwt_token = utils.get_jwt_token(
+        args.auth_url, args.app_client_id, args.app_client_secret
+    )
+    if not jwt_token:
+        logging.error("Failed to get JWT token.")
+        sys.exit(1)
+
+    response = Optional[str]
+
+    response, status = create_api_key.create_apikey(
+        args.customer_id, args.corpus_id, jwt_token
+    )
+    logging.info("CreateApiKey response: %s, status: %s", response, status)
+
+    if not status:
+        sys.exit(1)
+
+    api_key: str = response
```

Comment:
This fails mypy checking (run `mypy --namespace-packages -p api_key1` from the `python` directory).

Remove the `str =` typing here and remove the `Optional` on the first entry in the `create_apikey` return value.

Reply:
I was getting a mypy error, that's why I added it. The error was:
`api_key/api_key_main.py:61: error: Need type annotation for "api_key"  [var-annotated]`

After adding this annotation, mypy error for me went away. I was running it on a single file like following.
`mypy api_key/api_key_main.py `